### PR TITLE
Infra proxy nodes list UI

### PR DESCRIPTION
**Pull request by vinay033**
### :nut_and_bolt: Description: What code changed, and why?
we need to show the list of the infra proxy nodes under the org details.

### :chains: Related Resources
https://github.com/chef/automate/issues/4371
### :+1: Definition of Done
Added nodes tabs for the org where we can show the list of the infra proxy nodes.

### :athletic_shoe: How to Build and Test the Change
```
STEP 1
inside the hab studio

[default:/src:0]# build components/automate-ui-devproxy/
[default:/src:0]# start_automate_ui_background
[default:/src:0]# start_all_services

STEP 2
open new window
go to automate UI path

$ cd components/automate-ui
and run the command 

$npm run serve:hab
```
### :white_check_mark: Checklist

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [ ] Docs added/updated?
- [ ] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### :camera: Screenshots, if applicable
infra-nodes

### CODE REVIEW

**Comment by shaik80** [comment]
it look's good to me 🎉

File: components/automate-ui/src/app/entities/infra-nodes/infra-nodes.requests.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { NodesSuccessPayload } from './infra-nodes.actions';
+import { InterceptorSkipHeader } from 'app/services/http/http-client-auth.interceptor';
+
+@Injectable()
+export class InfraNodeRequests {
+
+  constructor(private http: HttpClient) { }
+
+  public getNodes(server_id: string, org_id: string):
+  Observable<NodesSuccessPayload> {
```
Comment (msorens) [inline components/automate-ui/src/app/entities/infra-nodes/infra-nodes.requests.ts:14]: Let's only break a line if necessary.
```suggestion
  public getNodes(server_id: string, org_id: string): Observable<NodesSuccessPayload> {
```

File: components/automate-ui/src/app/entities/infra-nodes/infra-nodes.requests.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    const headers = new HttpHeaders().set(InterceptorSkipHeader, '');
```
Comment (msorens) [inline components/automate-ui/src/app/entities/infra-nodes/infra-nodes.requests.ts:15]: When is the `InterceptorSkipHeader` needed? Or to look at it another way, why does it seem to be needed in many of the infra calls, but not in other calls?

File: components/automate-ui/src/app/modules/infra-proxy/infra-nodes/infra-nodes.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(InfraNodesComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```
Comment (msorens) [inline components/automate-ui/src/app/modules/infra-proxy/infra-nodes/infra-nodes.component.spec.ts:54]: Can we add some real unit tests to this file?

**Comment by SEAjamieD** [comment]
Is this supposed to be in the Ready for Review column?

**Pull request closed without merging**